### ISSUE TITLE
Stream error

### ISSUE THREAD

**Issue by Seanny123** [ownaginatious/fbchat-archive-parser#20]
I'm not getting the following error on both Python 2 and 3 on the master branch:
```
Traceback (most recent call last):                                                            
  File "/usr/local/bin/fbcap", line 11, in <module>
    load_entry_point('fbchat-archive-parser', 'console_scripts', 'fbcap')()
  File "/home/sean/git/fbchat-archive-parser/fbchat_archive_parser/main.py", line 156, in main
    app.run()
  File "/usr/local/lib/python3.5/dist-packages/clip.py", line 652, in run
    self.invoke(self.parse(tokens))
  File "/usr/local/lib/python3.5/dist-packages/clip.py", line 634, in invoke
    self._main.invoke(parsed)
  File "/usr/local/lib/python3.5/dist-packages/clip.py", line 519, in invoke
    self._callback(**{k: v for k, v in iteritems(parsed) if k not in self._subcommands})
  File "/home/sean/git/fbchat-archive-parser/fbchat_archive_parser/main.py", line 100, in fbcap
    write(format, fbch)
TypeError: write() missing 1 required positional argument: 'stream'
```

Don't know if you want to make the stream argument optional or what exactly is going wrong here.

**Comment by ownaginatious** [comment]
Yeah... stupid mistake on my part. Forgot to test everything fully (only checked that parsing started) after changing things to make the unit testing of the time stamp parsing easier. Fixed. Thanks 👍

**Comment by ownaginatious** [comment]
Fixed in `0.8.post9`.
